clang-tools-extra/clangd/CodeComplete.cpp

```diff
@@ -2122,7 +2125,10 @@ clang::CodeCompleteOptions CodeCompleteOptions::getClangCompleteOpts() const {
   // When an is used, Sema is responsible for completing the main file,
   // the index can provide results from the preamble.
   // Tell Sema not to deserialize the preamble to look for results.
-  Result.LoadExternal = !Index;
+  //
+  // FIXME: If we're using C++20 modules, force the lookup process to load external decls,
+  // since currently the index doesn't support C++20 modules.
```

Comment:
i think this comment belongs to `ClangdMain.cpp` instead.

Reply:
Done